### PR TITLE
controllers and climb command

### PR DESCRIPTION
**Pull request by laviRZ**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -79,19 +93,40 @@ private void initializeSubsystems() {
...
         swerveSS.setDefaultCommand(driveWithXboxCMD);
+
         driverXbox.getYBtn().whenPressed(new InstantCommand(swerveSS::resetGyro));
         driverXbox.getRightBumperBtn().whileHeld(new ParallelCommandGroup(intakeCMD, transportCMD));
         driverXbox.getLeftBumperBtn().whenPressed(new InstantCommand(intakeOpenerSS::toggleState));
+
+        driverXbox.getLeftBumperBtn().whileHeld(new ParallelCommandGroup(inverseIntakeCMD, inverseTransportCMD));
+        commanderXbox.getXBtn().whenPressed(new InstantCommand(() -> {
+            climbCMD.cancel();
+        }));
```
Comment (lycaon7) [inline src/main/java/frc/robot/RobotContainer.java:126]: climbCMD.withInterupt(getX())

File: src/main/java/frc/robot/subsystems/climber/ClimbCMD.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    public void initialize() {
+        climberSS.setSetpoint(pos);
+    }
+
+    @Override
+    public void execute() {
+
+    }
```
Comment (lycaon7) [inline src/main/java/frc/robot/subsystems/climber/ClimbCMD.java:27]: del

File: src/main/java/frc/robot/subsystems/shooter/PitcherSS.java [hunk 3]
```diff
@@ -72,7 +73,10 @@ public void initSendable(SendableBuilder builder) {
 
         builder.addDoubleProperty("Stats/setpoint", () -> Math.round(getSetpoint()), this::setSetpoint);
 
-        builder.addBooleanProperty("Reset Encoder CMD", () -> false, (x) -> resetEncoder());
+        builder.addBooleanProperty("Reset Encoder", () -> false, (x) -> {
+            if(x)
+                resetEncoder();
```
Comment (lycaon7) [inline src/main/java/frc/robot/subsystems/shooter/PitcherSS.java:78]: tuning?

**Review by lycaon7 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/shooter/PitcherSS.java (hunk 3 above)
Comment (laviRZ) [reply src/main/java/frc/robot/subsystems/shooter/PitcherSS.java:78]: nah

**Pull request merged**